cmd/fs-v1.go

```diff
@@ -103,6 +103,11 @@ func newFSObjectLayer(fsPath string) (ObjectLayer, error) {
 		}
 	}
 
+	// Check if disk has minimum required total space.
+	if err = checkDiskTotalMin(fsPath); err != nil {
+		return nil, err
+	}
```

Comment:
@krishnasrinivas this function uses Total disk space. checkDiskFree for free space only. If you use this during the startup what can happen is if a Disk i.e 100GB and upto 95% full can lead to `minio server` not starting at all. 

Total disk is needed so that we know we can't even have 1GB worth of free space anyways.